### ISSUE TITLE
Consider adding @none.com to known invalid domains

### ISSUE THREAD

**Issue by benrugg** [maximeg/email_inquire#7]
First of all, thank you for this gem. I've been using it for a while now, and it's super helpful.

Would you consider adding none.com to the known invalid domains? (I get users putting this as their email sometimes).

**Comment by maximeg** [comment]
Thanks 😀

In known invalid domains we have only domains for which we know for sure that they won't have email addresses, such as IANA's `example.com`.
`none.com` being a valid domain, I a bit reluctant to add it to the known invalid list.

I added support for Custom invalid domains so that one can add its own bad domains:
```
# in config/initializers/email_inquire.rb
EmailInquire.custom_invalid_domains << "none.com"
```
That was something on my todo list :)

**Comment by maximeg** [comment]
Change included in v0.8.0 released just now.

**Comment by benrugg** [comment]
Yeah, that makes a lot of sense. Thanks for adding the custom domains feature - that's a great addition!
